packages/E2ETest/package.json

```diff
@@ -51,7 +49,7 @@
     "ts-node": "^7.0.1",
     "tsconfig-paths": "^3.8.0",
     "typescript": "^3.8.3",
-    "webdriver": "git+https://github.com/react-native-windows/webdriver.git",
-    "webdriverio": "5.12.1"
+    "webdriverio": "^6.9.0",
```

Comment:
>"webdriverio": "^6.9.0", [](start = 4, length = 24)

wdio-winappdriver-service currently has a devDependency on webdriverio version ^6.10.0.  Should that be changed to a real dependency and this dependency removed?  Right now it's essentially a peer dependency, but that isn't specified in wdio-winappdriver

Reply:
We don't think we care about devDependency of wdio-winappdriver-service because it's not a real dependency to the RNW project

Here is yarn list
```
├─ wdio-winappdriver-service@0.1.0
│  ├─ @wdio/logger@^6.8.0
│  ├─ fs-extra@^9.0.1
│  ├─ fs-extra@9.0.1
│  │  ├─ at-least-node@^1.0.0
│  │  ├─ graceful-fs@^4.2.0
│  │  ├─ jsonfile@^6.0.1
│  │  └─ universalify@^1.0.0
│  ├─ jsonfile@6.1.0
│  │  ├─ graceful-fs@^4.1.6
│  │  ├─ universalify@^2.0.0
│  │  └─ universalify@2.0.0
│  └─ universalify@1.0.0
```

---
In reply to: [539641592](https://github.com/microsoft/react-native-windows/pull/6636#discussion_r539641592) [](ancestors = 539641592)

Reply:
Sorry if I wasn't clear.  I was suggesting that we consider changing wdio-winappdriver-service to declare a real dependency on webdriverIO, as this service appears to require webdriver to function correctly (I implied this via the name of the package, which has wdio in it.  

This would allow RNW to not have to declare a real dependency on webdriverIO itself, as it would pick it up through wdio-winappdriver-service.

Feel free to discuss offline.

---
In reply to: [540400945](https://github.com/microsoft/react-native-windows/pull/6636#discussion_r540400945) [](ancestors = 540400945,539641592)